MyJobApplication/src/com/amaderorg/myjobapp/Model/Database/ApplicationContentProvider.java

```diff
@@ -0,0 +1,107 @@
+package com.amaderorg.myjobapp.Model.Database;
+
+import android.content.ContentProvider;
+import android.content.ContentUris;
+import android.content.ContentValues;
+import android.content.UriMatcher;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteQueryBuilder;
+import android.net.Uri;
+
+import com.amaderorg.myjobapp.Model.Database.Tables.ContactsTableSchema;
+
+/**
+ * Created by souvi_000 on 3/30/2016.
+ */
+public class ApplicationContentProvider extends ContentProvider {
+    private static SQLiteDatabase mSQLiteDatabase;
```

Comment:
Added documentations for variables and non-overridden methods. Documents for overridden methods can be found in their corresponding Interface or abstract classes.